empyrical/stats.py

```diff
@@ -727,6 +731,44 @@ def tail_ratio(returns):
         np.abs(np.percentile(returns, 5))
 
 
+def cagr(returns, period=DAILY, annualization=None):
+    """
+    Compute compound annual growth rate.
+
+    Parameters
+    ----------
+    returns : pd.Series
+        Daily returns of the strategy, noncumulative.
+        - See full explanation in :func:`~empyrical.stats.cum_returns`.
+    period : str, optional
+        Defines the periodicity of the 'returns' data for purposes of
+        annualizing. Value ignored if `annualization` parameter is specified.
+        Defaults are:
+            'monthly':12
+            'weekly': 52
+            'daily': 252
+    annualization : int, optional
+        Used to suppress default values available in `period` to convert
+        returns into annual returns. Value should be the annual frequency of
+        `returns`.
+        - See full explanation in :func:`~empyrical.stats.annual_return`.
+
+    Returns
+    -------
+    float, np.nan
+        The CAGR value.
+
+    """
+    if len(returns) < 1:
+        return np.nan
+
+    ann_factor = annualization_factor(period, annualization)
+    no_years = float(len(returns)/ann_factor)
```

Comment:
I think you need float() of either numerator or denominator, but not the result.